src/app.js

```diff
@@ -56,7 +56,25 @@ export function app(app) {
     })
   }
 
+  function hydrate(elm) {
```

Comment:
Well a vnode function would return a vnode like { tag, data, children }

```js
function vnode(tag, data, children) {
  return { tag: tag, data: data, children: children }
}
```

Now we can reuse it in h and app. Only makes sense if we use more than once, and it seems we will. Don't know if this would save any byte, but it's a good practice. 

Can be done later.